src/pages/login/login.ts

```diff
@@ -106,8 +109,27 @@ export class LoginPage extends BasePageComponent {
 
       if (error === AuthenticationError.USER_CANCELLED) {
         this.analytics.logException(error, true);
+        const log: Log = this.createLog(LogType.INFO,
+          `user cancelled login`);
+        this.store$.dispatch(new SaveLog(log));
+        this.store$.dispatch(new SendLogs());
       }
       if (error === AuthenticationError.USER_NOT_AUTHORISED) {
+
+        const token = await this.authenticationProvider.getAuthenticationToken();
+        const examiner = await this.authenticationProvider.getEmployeeId() || '';
```

Comment:
Perhaps change this to say 'unavailable' so the log is more intelligible?

Reply:
changed.

Reply:
done